CromIAM/src/main/scala/cromiam/sam/SamClient.scala

```diff
@@ -23,13 +23,22 @@ import scala.concurrent.{ExecutionContextExecutor, Future}
      out into workbench-libs. We should replace this with that once the stars line up but for now it doesn't seem
      worth it. If one finds themselves making heavy changes to this file, that statement should be reevaluated.
  */
-class SamClient(scheme: String, interface: String, port: Int, log: LoggingAdapter, serviceRegistryActorRef: ActorRef)
+class SamClient(scheme: String,
+                interface: String,
+                port: Int,
+                checkSubmitWhitelist: Boolean,
+                log: LoggingAdapter,
+                serviceRegistryActorRef: ActorRef)
                (implicit system: ActorSystem, ece: ExecutionContextExecutor, materializer: ActorMaterializer) extends StatusCheckedSubsystem with CromIamInstrumentation {
 
   override val statusUri = uri"$samBaseUri/status"
   override val serviceRegistryActor: ActorRef = serviceRegistryActorRef
 
   def isSubmitWhitelisted(user: User, cromIamRequest: HttpRequest): Future[Boolean] = {
+    if (!checkSubmitWhitelist) {
+      return Future.successful(true)
+    }
+
```

Comment:
Any reason not to `if/else` this logic to avoid the `return`?

Reply:
Reason, sure? But not the strongest one. Using `else` means indenting the rest of the function. Which then increases the size of the diff.